setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = python-blazarclient
+name = chameleon-blazarclient
```

Comment:
Does this change the `module` names, for things that might `import blazarclient`

And I was trying to figure out what happens if you install both `python-blazarclient` and `chameleon-blazarclient` into the same venv

And I think pip will actually just partially overwrite one of them:
https://github.com/pypa/pip/issues/4625

Reply:
well, all the unittests still pass, so I think`from blazarclient import foo` still works